kawin/Thermodynamics.py

```diff
@@ -49,20 +49,26 @@ class GeneralThermodynamics:
     gOffset = 1      #Small value to add to precipitate phase for when order/disorder models are used
 
     def __init__(self, database, elements, phases, drivingForceMethod = 'approximate'):
-        self.db = Database(database)
+        if type(database) == str:
```

Comment:
Slightly better to use `isinstance(database, str)` here.

Reply:
Thanks, I see that it is a better method to check the type.